package.json

```diff
@@ -47,5 +47,8 @@
   },
   "scripts": {
     "dev-server": "node_modules/.bin/webpack-dev-server --content-base docs/ --inline --watch"
+  },
+  "dependencies": {
+    "underscore": "^1.8.3"
```

Comment:
if possible, would really like to not include underscore as a dependency in this lib